docs/reference/search/validate.asciidoc

```diff
@@ -202,7 +263,8 @@ GET twitter/_validate/query?rewrite=true&all_shards=true
 }
 --------------------------------------------------
 
-Response:
+
```

Comment:
The prior snippet needs `// TEST[continued]`.

Reply:
@jrodewig Thank you so much, it solved the problem!